README.md

```diff
@@ -19,20 +19,26 @@
 <!-- ABOUT THE PROJECT -->
 ## About The Project
 
-Draft makes it easier for developers to get started building apps that run on Kubernetes by taking a non-containerized application and generating the Dockerfiles, Kubernetes manifests, Helm charts, Kustomize configuration, and other artifacts associated with a containerized application. Draft can also generate a GitHub Action workflow file to quickly build and deploy applications onto any Kubernetes cluster.   
+Draft makes it easier for developers to get started building apps that run on Kubernetes by taking a non-containerized application and generating the Dockerfiles, Kubernetes manifests, Helm charts, Kustomize configuration, and other artifacts associated with a containerized application. Draft can also generate a GitHub Action workflow file to quickly build and deploy applications onto any Kubernetes cluster.
 
-* `draft create` adds the minimum required files for your deployment to the project directory.
+### Commands
+
+* `draft create` adds the minimum required Dockerfile and manifest files for your deployment to the project directory.
+  * Supported deployment types: Helm, Kustomize, Kubernetes manifest.
 * `draft setup-gh` automates the Github OIDC setup process for your project.
-* `draft generate-workflow` generates a Github workflow for automatic build and deploy to AKS.
-* `draft update` automatically updates your application to be internet accessible.
+* `draft generate-workflow` generates a Github Action workflow for automatic build and deploy to a Kubernetes cluster.
```

Comment:
💡☕️ question: day 0 does this tool support all flavours of Kubernetes cluster? Asking because you removed AKS.

Reply:
The dockerfile and deployment generation is provider agnostic

Reply:
Cool, thanks @imiller31 ! I guess what I was trying to point out that is this file is part of the workflow for example https://github.com/actions/starter-workflows/blob/main/deployments/azure-kubernetes-service-helm.yml then right at the top it says AKS specificity part.☕️Let me ping you in teams as well.